index.js

```diff
@@ -115,6 +118,21 @@ cm.Item({
   }
 });
 
+cm.Item({
+  label: 'Send to mini player',
+  context: cm.SelectorContext('[href*="vine.co/v/"]'),
+  contentScript: 'self.on("click", function (node, data) {' +
+                 '  self.postMessage(node.href);' +
+                 '});',
+  onMessage: function(url) {
+    const id = getVideoId(url);
+    updatePanel({domain: 'vine.co', id: id, src: ''});
+    getVineUrl(url, function(err, streamUrl) {
+      if (!err) updatePanel({src: streamUrl});
```

Comment:
Good catch! Actually `getVineUrl` never returns an error message, so we never need to handle it ;-)